sw/device/lib/testing/edn_testutils.h

```diff
@@ -9,14 +9,19 @@
 
 /**
  * Returns randomized seed material.
+ *
```

Comment:
Here you should probably add a description of the `disable_rand` parameter.

Reply:
Nice catch, thanks!